validphys2/src/validphys/results.py

```diff
@@ -514,7 +524,7 @@ def results(dataset: (DataSetSpec), pdf: PDF, covariance_matrix, sqrt_covmat):
     # probably not in most cases...
     return (
         DataResult(dataset, covariance_matrix, sqrt_covmat),
-        ThPredictionsResult.from_convolution(pdf, dataset),
+        ThPredictionsResult.from_convolution(pdf, dataset, central_only=central_only),
```

Comment:
```suggestion
        ThPredictionsResult.from_convolution(pdf, dataset),
```

You can use `results_central` which is defined below.